### DISCUSSION TITLE
青铜git勋章怎么获得呢

### DISCUSSION THREAD

**Discussion by selltk** [ByteLegendQuest/remember-brave-people#495] [Q&A]
没搞懂这个勋章怎么获得

**Comment by blindpirate** [comment]
正在开发中，请耐心等待。

**Comment by selltk** [comment]
好的感谢



------------------&nbsp;原始邮件&nbsp;------------------
发件人: "Bo ***@***.***&gt;; 
发送时间: 2021年9月21日(星期二) 晚上6:25
收件人: ***@***.***&gt;; 
抄送: ***@***.***&gt;; ***@***.***&gt;; 
主题: Re: [ByteLegendQuest/remember-brave-people] 青铜git勋章怎么获得呢 (Discussion #495)






正在开发中，请耐心等待。

—
您正在接收此消息，因为您创建了该线程。
直接回复此邮件，在 GitHub 上查看或取消订阅.
GitHub Mobile 上的分流通知监督办或机器人.
